tests/e2e/test_client_job_ingestion.py

```diff
@@ -284,7 +333,7 @@ def test_post_job_artifacts_by_add_artifact(
         result_set_stored,
         mock_post_collection,
         mock_error_summary,
-        text_log_summary_blob,
+        text_log_summary_dict,
```

Comment:
this one actually is referenced.  But there was another one that wasn't.  So I removed it.